Header header
float64[3] standard_deviation
float64[3] position
int16[2] self_observation
int16[2] other_observation
